scheduler/src/cook/kubernetes/api.clj

```diff
@@ -233,8 +233,8 @@
     (.setEnv container (into [] env))
     (.setImage container image)
 
-    (.putRequestsItem resources "memory" (double->quantity (* 1024 1024 mem)))
-    (.putLimitsItem resources "memory" (double->quantity (* 1024 1024 mem)))
+    (.putRequestsItem resources "memory" (double->quantity (* 1000000 mem)))
```

Comment:
"Cook tracks memory in megabytes (1000*1000) bytes. Convert its megabytes into bytes."

Reply:
Done